src/main/java/brave/opentracing/BraveSpanBuilder.java

```diff
@@ -105,10 +111,32 @@
     return this;
   }
 
+  @Override
+  public Span start() {
+    return startManual();
+  }
+
+  @Override
+  public ActiveSpan startActive() {
+    if (!ignoreActiveSpan) {
+      ActiveSpan parent = tracer.activeSpan();
+      if (parent != null) {
+        asChildOf(parent);
+      }
+    }
+    return tracer.makeActive(startManual());
+  }
+
+  @Override
+  public SpanBuilder ignoreActiveSpan() {
+    ignoreActiveSpan = true;
+    return this;
+  }
+
   /**
    * {@inheritDoc}
    */
-  @Override public BraveSpan start() {
```

Comment:
the covariant override w/BraveSpan was there so that folks could unwrap into brave w/o casting.